src/App.js

```diff
@@ -11,6 +10,8 @@ const App = () => {
         <Route path="/products" element={<ProductListing />} />
```

Comment:
hey u can make a separate file for routes, it is better to keep App as clean as possible

Reply:
Ok, I will do that. Thanks!!